src/content/en/tools/workbox/guides/_shared/register-sw.md

```diff
@@ -0,0 +1,12 @@
+{# wf_md_include #}
+
+<pre class="prettyprint html">
+&lt;script&gt;
```

Comment:
It looks like there's only an opening `<script>` without a closing tag. Unless that gets inserted by another partial template?